dataprep/eda/utils.py

```diff
@@ -450,8 +451,9 @@ def _format_axis(fig: Figure, minv: int, maxv: int, axis: str) -> None:
 
     # make the tick values
     ticks = [float(minv)]
-    while max(ticks) + gap < maxv:
-        ticks.append(max(ticks) + gap)
+    if gap == np.inf or int(minv) != int(maxv):
```

Comment:
The same comment as on use of `np.isinf`. Besides, when `gap` is `inf`, will we also go to the `while` loop?